diff --git a/src/core/lib/gpr/wrap_memcpy.cc b/src/core/lib/gpr/wrap_memcpy.cc
index 9b8608e056..4e165df8fd 100644
--- a/src/core/lib/gpr/wrap_memcpy.cc
+++ b/src/core/lib/gpr/wrap_memcpy.cc
@@ -29,7 +29,7 @@
 extern "C" {
 #ifdef __linux__
 #if defined(__x86_64__) && !defined(GPR_MUSL_LIBC_COMPAT)
-__asm__(".symver memcpy,memcpy@GLIBC_2.2.5");
+// __asm__(".symver memcpy,memcpy@GLIBC_2.2.5");
 void* __wrap_memcpy(void* destination, const void* source, size_t num) {
   return memcpy(destination, source, num);
 }
